_acs-aem-commons/features/packagers/asset-packager/index.md

```diff
@@ -0,0 +1,66 @@
+---
+layout: acs-aem-commons_feature
+title: Asset Packager
+description: Easily package pages and the associated assets
+redirect_from: /acs-aem-commons/features/asset-packager.html
+date: 2019-05-20
+feature-tags: administration
+initial-release: 4.2.0
+---
+
+# Purpose
+
+The Asset Packager lets you create a content package that is given a specific page path and will iterate over the content of that package to add all referenced assets into the package as well, giving you exactly what you need for a single page with all images.
+
+Helps a developer looking to:
+* Replicate an issue specific to content on a remote server
+* Not bloat your local instance unnecessarily
```

Comment:
Bullets 2 and 3 here feel like sub-bullets to bullet 1.  Then the "or just a developer looking to get set up ..." line below can become a bonified bullet since it is indeed a use case like bullet 1 is.

Reply:
Updated in the latest commits